crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -264,7 +264,7 @@ where
                     {
                         Ok(network) => network,
                         Err(err) => {
-                            panic!("Failed to create network: {err}");
+                            panic!("Failed to create network: {err:?}");
```

Comment:
Nit: Can we make this "Failed to create libp2p network" - just to make it easier to understand if it shows up :)

Reply:
LGTM otherwise!